backend/src/main/java/com/google/rolecall/Constants.java

```diff
@@ -32,16 +32,25 @@
 
   public static class Roles {
```

Comment:
Naming-wise I mirrored exactly what was done with Permissions (including the names for all supporting functions). If this is a trivial change, we should consider it. Otherwise it should be twice the effort because we should do Permissions as well.

Reply:
Ok, let's leave it as-is for now.